doc/index.md

```diff
@@ -1,11 +1,111 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.6
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+<!-- #raw -->
+<!-- 
+The original figure in this documentation were produced with Xournal++ (./files/others/example_plot.xopp). The content of the XOPP was exported as pdf (./files/others/example_plot.pdf). With the paginate function of svgdigitizer the page was exported as PNG and SVG.
+* (./files/others/example_plot.png)
+* (./files/others/example_plot.svg)
+
+The latter was renamed and to ./files/others/example_plot_demo.svg and the curve was digitized. The digitized plot was exported as ./files/others/example_plot_demo.png
+-->
+<!-- #endraw -->
+
```

Comment:
@saraedum I included the workflow on how the documentation has been built as a comment in the file. Then all unused files are at least mentioned somewhere. I think this is a rather transparent approach.

Some of these files will presumably be recycled in the other docu.